packages/tools/src/utilities/dynamicVolume/getDataInTime.ts

```diff
@@ -0,0 +1,104 @@
+import { utilities, cache } from '@cornerstonejs/core';
+
+/**
+ *
+ * @param dynamicVolume: 4D volume to compute time point data from
+ * @param options: frameNumbers: which frames to us as timepoints, if left
+ * blank, gets data timepoints over all frames
+ * maskVolumeId: segmentationId to get timepoint data of
+ * imageCoordinate: image coordinate to get timepoint data of
+ * @returns
+ */
+function getDataInTime(
+  // dynamicVolumeId: string,
+  dynamicVolume: any,
```

Comment:
this shouldn't be of type `any` it should be type of the dynamic volume

Reply:
changed to IDynamicImageVolume, removed commented out dynamicVolumeId